fss_check/task_schedule.cfg

```diff
@@ -37,6 +37,9 @@ alert       aca@head.cfa.harvard.edu
       exec fss_check_daily_fss --out $ENV{SKA}/data/fss_check3 --email=aca_alert\@cfa.harvard.edu
       exec /bin/cp -p $ENV{SKA}/data/fss_check3/index.html /data/mta4/www/ASPECT/fss_check3/
       exec /bin/cp -p $ENV{SKA}/data/fss_check3/*.png /data/mta4/www/ASPECT/fss_check3/
+      exec fss_check_daily_fss --days-recent=360 --out $ENV{SKA}/data/fss_check3/days-recent-360
+      exec /bin/cp -p $ENV{SKA}/data/fss_check3/days-recent-360/index.html /data/mta4/www/ASPECT/fss_check3/days-recent-360/
```

Comment:
Oh, and while we are in here, these output dirs should just be ska-relative too (${SKA}/www/fss_check3).  Hence the permissions error I just got when I tried to run a test.

Reply:
Done.